docs/src/documentation/03-components/04-overlay/popover/03-accessibility.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/popover/accessibility/
+---
+
+# Accessibility
+
+## Popover
+
+The Popover component has been designed with accessibility in mind, providing features that enhance the experience for users of assistive technologies.
+
+### Accessibility props
+
+The following props are available to improve the accessibility of your Popover component:
+
+| Prop           | Type                                                  | Default    | Description                                                                           |
+| :------------- | :---------------------------------------------------- | :--------- | :------------------------------------------------------------------------------------ |
+| ariaLabel      | `string`                                              |            | Adds `aria-label` to the popover content, providing a description for screen readers. |
+| ariaLabelledby | `string`                                              |            | References the ID of an element that labels the popover content for screen readers.   |
+| role           | `"dialog" \| "menu" \| "grid" \| "listbox" \| "tree"` | `"dialog"` | Defines the ARIA role of the popover component.                                       |
+| labelClose     | `React.Node`                                          | `"Close"`  | The label for the close button displayed on mobile devices.                           |
+
+### Automatic ARIA attributes
+
+The Popover component automatically handles several important ARIA attributes on the trigger element:
+
+- `aria-controls`: Links the trigger element to the popover content using a unique ID or `id` provided via props.
+- `aria-haspopup`: Indicates that the trigger reveals additional content (set to the popover's role).
+- `aria-expanded`: Indicates whether the popover is currently visible or hidden.
+
+These attributes are managed by the component and do not need to be manually added to your code.
+
+### Best practices
+
+#### Non-interactive trigger elements
+
+The Popover's trigger element (provided as `children`) should not be an interactive element like a native `<button>` or `<a>` tag, as this creates nested interactive elements which is an accessibility violation. Instead:
```

Comment:
This `Instead:` and note with a bullet point below look confusing to me.

Reply:
I can see how. There were multiple examples but incorrect so I removed them and now it doesn't make sense 😅 I'll rewrite.